### ISSUE TITLE
Reconcile->QA Facets

### ISSUE THREAD

**Issue by ostephens** [OpenRefine/OpenRefine#1370]
There are three options under Reconcile->QA Facets - these seem to me to be a hangover from the 'QA' concept in Freebase.

There are three options here - from what I can see one doesn't do anything and the other two could be moved into the more general Reconcile->Facets menu - as follows:

- QA Results - I don't think this does anything (it facets on cell.recon.features.qaResult but I can't see anything that populates that property anywhere?). Suggest removed
- Judgment Actions - could be moved to Reconcile->Facets menu. Suggest renamed 'Judgment Action type'
- Judgment History entries - could be moved to Reconcile->Facets menu. Suggest renamed 'Judgment Action Timestamp'

Any views on this?

**Comment by wetneb** [comment]
* QA results - I agree this is obsolete and should be removed. Does anybody know what sort of things `cell.recon.features.qaResult` used to store? @thadguidry?
* The two label changes look sensible to me (except the case: only the first word should be capitalized to keep consistency with the rest).

**Comment by thadguidry** [comment]
@wetneb qaResult displayed the gathered results from Freebase staff and expert judgments once a client recon data was uploaded to Freebase.  

```2.0 Release  (November 10, 2010 - first release as Google Refine)

Major Changes:
- New extension architecture.
- Generalized reconciliation framework that allows plugging in standard reconciliation services.
- Support for QA on data loads into Freebase.
- Timeline Facet (Issue 40 and 95)
```

**Comment by ostephens** [comment]
Fixed by #1373
